ground_station/src/main.cpp

```diff
@@ -85,12 +85,35 @@ void loop() {
     navigation.begin();
   }
 
+  // Update the home location
   if (link2.location.isUpdated()) {
```

Comment:
why only check link2 location here?

Reply:
This is not the location from link2, it is the gps that is connected to the link2 MCU. A bit confusing I know